settings.gradle

```diff
@@ -12,26 +13,30 @@ if (!JavaVersion.current().java11) {
 }
 
 gradle.ext.ciBuild = System.getenv()['JENKINS_URL'] ? true : false
```

Comment:
should this now be better named with Jenkins in it ?

Reply:
Renamed